benchmark.go

```diff
@@ -66,6 +67,13 @@ type Summary struct {
 	RPS                float64 `json:"rps"`
 }
 
+// ErrorSummary stores the summary of the errors encountered
+type ErrorSummary struct {
+	TotalErrors int     `json:"totalErrors"`
+	ErrorRate   float64 `json:"errorRate"`
+	ErrorsCount map[string]int
```

Comment:
nit: can you add the JSON tag here as well?

Reply:
Good point! Fixed.